test/test_parse.py

```diff
@@ -1789,6 +1789,45 @@ def test_names(r):
     assert s.tree is not None
     assert s.tree.persons == []
     assert s.tree.nouns == ["sýning", "bíó", "þriðjudagskvöld"]
+    s = r.parse_single("Ruud van Nistelrooy og Thomas de Broglie komu í heimsókn.")
+    assert "Thomas de Broglie" in s.tree.persons
+    # "Ruud van Nistelrooy" gets interpreted as an entity
+    # assert "Ruud van Nistelrooy" in s.tree.persons
```

Comment:
We could still check for 'Ruud van Nistelrooy' as a whole (entity) token in the test, I recommend doing that.

Reply:
Added entity check!